regolith/helpers/makeappointmentshelper.py

```diff
@@ -102,13 +103,16 @@ def sout(self):
             is_fully_appointed(person, this_begin, this_end)
             for appt in appts:
                 if appt.get("grant") in BLACKLIST:
+                    print(appt.get("grant"))
                     continue
                 grant = rc.client.find_one(rc.database, "grants", {"_id": appt.get("grant")})
+                prop = rc.client.find_one(rc.database, "proposals", {"_id": grant.get("proposal_id")})
+                grant_begin, grant_end = get_dates(prop)['begin_date'], get_dates(prop)['end_date']
```

Comment:
I think this assumes that the begin and end date will be in the proposal, but we only want to get it from the proposal if it is not found in the grant.  Sometimes these dates may be overwritten in the grant and what is in the grant should take precendent.  So only go look for them if they are not there in grant.  This was the behavior that merge_collections had, so we may want to go back to using that if it is a well tested function (though right now it is maybe not working).  I don't want to hold this up in case that takes a long time though, I urgently need this helper....

Reply:
done! haven't used `merge_collections` here. I tested it by removing/ overwriting begin and end dates in some grants compared to their referenced proposals - however, I had to re-enter the grant dates in the schema so that the grant lister test would pass